ckanext/datagovuk/public/datagovuk.css

```diff
@@ -110,3 +102,103 @@ p.home-footer-text {
   line-height: 1.5;
 }
 
+.form-control {
+  margin-bottom: 14px;
+}
+
+.mt-10 {
+  margin-top: 10px;
+}
+
+.mb-10 {
+  margin-bottom: 10px;
+}
+
+.control-label.col-md-3 {
+  text-align: right;
+  padding-right: 0;
+}
+
+.control-full input[type=radio]{
+  width: auto;
+}
+
+.harvest-types label.radio {
+  margin-bottom: 0;
+}
+
+.harvest-types label.radio input{
+  top: -1px;
```

Comment:
I don't think we can. The first two positions come from the upstream ckan project and harvest extension, only this one is in the DGU extension. My approach throughout has been to not touch the original code but to work on the extension directly as an update will potentially overwrite any changes otherwise.

Reply:
Ahhhhh my bad. Resolving.